build.gradle

```diff
@@ -34,6 +34,7 @@ import org.opensearch.gradle.test.RestIntegTestTask
 
 buildscript {
     ext {
+        System.setProperty("OPENSEARCH_INITIAL_ADMIN_PASSWORD", "myStrongPassword123!")
```

Comment:
we should be setting this value based on versions. i.e. for 2.11 and lower, it should be `admin` else use `myStrongPassword123`. Tests for 2.11 and lower would fail otherwise as the integtest.sh is modified to followed the same conditional assignment